gematria/datasets/find_accessed_addrs_test.cc

```diff
@@ -161,5 +179,24 @@ TEST_F(FindAccessedAddrsTest, AccessFromRegister) {
                                  ElementsAre(0x10000, 0x20000))));
 }
 
+TEST_F(FindAccessedAddrsTest, ExegesisNoAccess) {
+  auto addrs_or_err = FindAccessedAddrsExegesis(R"asm(
+    mov r11, r12
+  )asm");
+  ASSERT_TRUE(static_cast<bool>(addrs_or_err));
```

Comment:
Does LLVM have a gmock matcher for `llvm::Expected` similar to `IsOkAndHolds`?

I'd expect it to have one, and if not, it might be worth adding one (eventually, not in this PR). This is a very convenient way to write tests.

Reply:
It does, specifically `ASSERT_THAT_EXPECTED`, but linking against the library that provides it, `@llvm-project//llvm:TestingSupport` currently is causing test failures during cleanup due to a pthread error. I've left a TODO for the time being.